core/server/common/src/main/java/alluxio/cli/validation/SecureHdfsValidationTask.java

```diff
@@ -58,19 +58,39 @@ public SecureHdfsValidationTask(String process) {
 
   @Override
   public boolean validate(Map<String, String> optionsMap) {
-    if (super.shouldSkip()) {
+    if (shouldSkip()) {
       return true;
     }
     if (!super.validate(optionsMap)) {
       return false;
     }
     if (!validatePrincipalLogin()) {
-      System.err.format("Principal login test failed.");
+      System.err.format("Principal login test failed.%n");
```

Comment:
if the principal login test will already print the root cause, do we still need to print this here?

Reply:
Done.